v2/installer.ps1

```diff
@@ -0,0 +1,574 @@
+#!/usr/bin/env pwsh
+Add-Type -AssemblyName System.Windows.Forms
+Add-Type -AssemblyName System.Drawing
+
+$BRTX_DIR = "$env:LOCALAPPDATA\graphics.bedrock"
+
+$T = Data {
+    # Default
+    ConvertFrom-StringData -StringData @'
+    package_name = BetterRTX
+    browse = Browse...
+    install = Install
+    install_instance = Install to Instance
+    install_pack = Install Preset
+    install_custom = Custom
+    uninstall = Uninstall
+    uninstalled = Uninstalled
+    copying = Copying
+    downloading = Downloading
+    success = Success
+    error = Error
+    error_invalid_file_type = Invalid file type. Please select a .mcpack file.
+    error_no_installations_selected = Please select at least one Minecraft installation.
+    error_copy_failed = Unable to copy to Minecraft installation.
+    setup = Setup
+    download = Download
+    launchers = Launchers
+    help = Help
+    backup = Backup
+    backup_instance_location = Select backup location for instance
+    create_initial_backup = Creating initial backup
+'@
+}
+$translationFilename = "installer.psd1"
+$localeDir = Join-Path -Path $BRTX_DIR -ChildPath "Localized"
+$localizedDataPath = Join-Path -Path $localeDir -ChildPath "$PsUICulture\$translationFilename"
+
+if (-not (Test-Path $localizedDataPath)) {
+    $localizedData = "https://raw.githubusercontent.com/BetterRTX/BetterRTX-Installer/main/v2/Localized/$PsUICulture/${translationFilename}"
+    try {
+        Invoke-WebRequest -Uri $localizedData -OutFile $localizedDataPath
+    }
+    catch {
+        if ($_ -like "*404*") {
+            Write-Host "Language `"$PsUICulture`" currently not available." -ForegroundColor Yellow
+            Write-Host "(Translation contributions welcome! https://github.com/BetterRTX/BetterRTX-Installer/blob/main/CONTRIBUTING.md)" -ForegroundColor Magenta
+        }
+        else {
+            Write-Host "Failed to download locale $PsUICulture data: $_"
+        }
+
+        # Fallback to local data during development/translation
+        $localLocaleDir = Join-Path -Path $PSScriptRoot -ChildPath "Localized"
+
+        if (Test-Path $localLocaleDir) {
+            Clear-Host
+            $localeDir = $localLocaleDir
+            Write-Debug "Using translations in `"$localeDir`""
+        }
+    }
+}
+
+Import-LocalizedData -BaseDirectory $localeDir -ErrorAction:SilentlyContinue -BindingVariable T -FileName $translationFilename
+
+$ioBit = Get-StartApps | Where-Object { $_.Name -eq "IObit Unlocker" }
+$hasSideloaded = (Get-AppxPackage -Name "Microsoft.Minecraft*" | Where-Object { $_.InstallLocation -notlike "C:\Program Files\WindowsApps\" }).Count -gt 0
```

Comment:
This line may need the same treatment as the error on line 161, not sure though

Reply:
Adding a wildcard like `WindowsApps*` could match something like `C:\Program Files\WindowsApps_WaitNoThisIsActuallySomeFunnyBusiness`

I think this error needs confirmation after rerunning the latest version.

Reply:
Still the same issue with the latest version, unless you pushed a new commit since the readme rewrite.